addons/optics/CfgJointRails.hpp

```diff
@@ -13,3 +13,35 @@ class asdg_OpticRail1913: asdg_OpticRail {
         ACE_optic_LRPS_PIP = 1;
     };
 };
+
+// Vanilla rails
+class SlotInfo;
+class CowsSlot: SlotInfo {
```

Comment:
Duplicated in https://github.com/acemod/ACE3/blob/18739c23ceeaba11d698906a149d161c82576af1/addons/optics/CfgWeapons.hpp#L312